lib/perl/Genome/Site/TGI/Command/SyncCaptureSets.pm

```diff
@@ -1,4 +1,4 @@
-package Genome::Site::TGI::CaptureSet::Command::Synchronize;
+package Genome::Site::TGI::Command::SyncCaptureSets;
```

Comment:
I think it's time to retire this script (and its Jenkins job)!  These are now created using `genome feature-list import` directly.